backend/btrixcloud/crawlconfigs.py

```diff
@@ -1369,24 +1377,54 @@ def init_crawl_config_api(
     @router.get("", response_model=PaginatedCrawlConfigOutResponse)
     async def get_crawl_configs(
         org: Organization = Depends(org_viewer_dep),
-        pageSize: int = DEFAULT_PAGE_SIZE,
+        page_size: Annotated[
+            int, Query(alias="pageSize", title="Page Size")
+        ] = DEFAULT_PAGE_SIZE,
         page: int = 1,
         # createdBy, kept as userid for API compatibility
-        userid: Optional[UUID] = None,
-        modifiedBy: Optional[UUID] = None,
-        profileid: Optional[UUID] = None,
-        firstSeed: Optional[str] = None,
+        user_id: Annotated[
+            Optional[UUID], Query(alias="userid", title="User ID")
+        ] = None,
+        modified_by: Annotated[
+            Optional[UUID], Query(alias="modifiedBy", title="Modified By User ID")
+        ] = None,
+        profile_id: Annotated[
```

Comment:
Maybe we go with 'profileIds' to be clear that this is a list, like 'tags', or even 'profiles'